index.html

```diff
@@ -24,7 +24,7 @@ <h2 class="secHeading">Adventure Time!! </h2>
 
         
             <div class="bg bg1">
-                <h2 class="desc">BIKING</h2>
+                <h2 class="desc" id="bike">BIKING</h2>
             </div>
             <p class="text">
                 Lorem ipsum dolor sit amet consectetur, adipisicing elit. Qui necessitatibus commodi alias? Veritatis, eligendi nisi. Amet exercitationem, recusandae dolore corrupti laborum aperiam repellendus natus voluptatem quam atque aliquam beatae. Odit deserunt nisi optio eveniet? Aliquam possimus molestiae, ullam illum accusamus autem accusantium! Voluptate laboriosam eum saepe ex veritatis est? Numquam eaque autem labore provident dolorum, velit omnis mollitia eos adipisci!
```

Comment:
Please change this Lorem ipsum description to something that is relevant to the heading that is Biking